providers/grafana-on-call/package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "@novu/grafana-on-call",
+  "version": "0.16.3",
```

Comment:
This version should be the same as the rest of the providers, v0.21.0

Reply:
okay, out of curiocity, whats the reason to keep all provider version same? also this is auto generated.

Reply:
I am not sure, that is a @LetItRock question

Reply:
upgraded to 0.21.0